src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -510,7 +510,7 @@ private static String humanReadableSize(long size) {
             }
             final JenkinsLocationConfiguration jlc = JenkinsLocationConfiguration.get();
 
-            out.println("  * Url:     " + (jlc != null ? ContentFilter.filter(filter, jlc.getUrl()) : "No JenkinsLocationConfiguration available"));
+            out.println("  * Url:     " + (ContentFilter.filter(filter, jlc.getUrl())));
```

Comment:
Why did you remove this Null test @Evildethow  ? (I don't know that code thus maybe it's useless..)

Reply:
`get` is `@Nonnull`.